tests/integration/backends/test_emis.py

```diff
@@ -153,3 +157,67 @@ def test_patients(select_all_emis):
 
 def test_registered_tests_are_exhaustive():
     assert_tests_exhaustive(EMISBackend())
+
+
+def test_inline_table_includes_organisation_hash(trino_database):
```

Comment:
I think there's probably a cleaner and simpler way of testing the thing you want here but I don't think that's worth worrying about this stage.

Reply:
I still think this, but on reflection I don't think we have to care about this right now. I'm satisfied the test _does_ test the thing it needs do. We can look at simplifying it later.

Reply:
I agree, but couldn't figure out the better way to do it (yet)